zio-http/src/main/scala/zhttp/http/Middleware.scala

```diff
@@ -176,6 +182,11 @@ object Middleware extends HttpMiddlewares {
    */
   def succeed[B](b: B): Middleware[Any, Nothing, Nothing, Any, Any, B] = fromHttp(Http.succeed(b))
 
+  /**
+   * Times out if execution is not completed within specified duration, with a None output
+   */
+  def timeout[B]: PartialTimeout[B] = new PartialTimeout[B](())
```

Comment:
I think we can drop the type param `B`

Reply:
Not Infering

Reply:
Let's drop this operator in that case. Until we are clear with the API signature we don't need to add it.